### DISCUSSION TITLE
Possible configuration issue...

### DISCUSSION THREAD

**Discussion by JohnBanas** [histoire-dev/histoire#306] [Q&A] [answered]
I am beyond excited to try out this project. The initial setup will render basic components just fine, but will not render any content that uses external libraries from npm. I assumed this is a configuration/setup process missing on my part.

// in histoire.config.ts
```ts
import { defineConfig } from "histoire";

export default defineConfig({
  setupFile: "@/histoire.setup.ts",
  plugins: [HstVue()],
});
```

// in histoire.setup.ts
```ts
import "@/styles/index.css";
import "floating-vue/dist/style.css";
import { createPinia } from "pinia";
import { defineSetupVue3 } from "@histoire/plugin-vue";
import VueTablerIcons from "vue-tabler-icons";
import router from "@/router";
import FloatingVue from "floating-vue";
import i18n from "@/plugins/i18n";

export const setupVue3 = defineSetupVue3(({ app, story, variant }) => {
  app.use(router);
  app.use(FloatingVue);
  app.use(i18n);
  app.use(createPinia());
  app.use(VueTablerIcons);
});

```

Still nothing renders in the individual stories if they contain one of these libraries. There are no errors either. Am I completely off on my setup here? Thanks very much for any feedback, or help.

**Comment by Akryum** [comment, marked as answer]
Do you have a runnable reproduction? Thanks!

**Reply by JohnBanas** [reply]
Never mind I got it working! Thank you for your quick response. It was the path to my setupFile, when I changed

  `setupFile: "@/histoire.setup.ts",`
  
  to 
  
  `setupFile: "./src/histoire.setup.ts",`
  
  everything worked as expected.
